ci/nightly/pipeline.yml

```diff
@@ -31,17 +31,17 @@
   label: ":panda_face: :racing_car: testdrive"
   plugins:
     - ./ci/plugins/mzcompose:
-        composition: redpanda
-        run: redpanda-testdrive
+        composition: testdrive
+        run: testdrive-redpanda-ci
```

Comment:
I do not think there is a `testdrive-redpanda-ci` anymore, I guess you need `args` here with `--redpanda`

Reply:
OH sorry my fault I spaced out.